### ISSUE TITLE
Install script 

### ISSUE THREAD

**Issue by ferricoxide** [aws/aws-cli#2167]
See master-branch [install script](https://github.com/aws/aws-cli/blob/master/scripts/install#L89-L94):

    def pip_install_packages(install_dir):
        cli_tarball = [p for p in os.listdir(PACKAGES_DIR)
                   if p.startswith('awscli')]
        assert len(cli_tarball) == 1
        cli_tarball = cli_tarball[0]

This fails horribly:

    # /tmp/awscli-bundle/install -i /opt/aws/cli -b /usr/bin/aws
    Running cmd: /bin/python virtualenv.py --python /bin/python /opt/aws/cli
    Traceback (most recent call last):
      File "/tmp/awscli-bundle/install", line 138, in <module>
        main()
      File "/tmp/awscli-bundle/install", line 129, in main
        pip_install_packages(opts.install_dir)
      File "/tmp/awscli-bundle/install", line 92, in pip_install_packages
        assert len(cli_tarball) == 1
    AssertionError

If your download somehow has more than one `awscli` tarfile found:

    [root@ip-172-31-13-25 awscli-bundle]# find . -name "awscli*"
    ./packages/awscli-1.10.62.tar.gz
    ./packages/awscli-1.10.63.tar.gz

Not sure how - since this was a fresh pull (08:48 Eastern, today) from the bucket to a newly-spun (us-west-1) instance - but I ended up with the 1.10.62 _and_ 1.10.63 tgzs in my bundle/bundle-dir.

**Comment by jamesls** [comment]
I'm not able to repro the issue.  I'm only seeing a single CLI tarball.  Here's what I tried:

```
$ cd /tmp/
$ wget aws-cli.s3.amazonaws.com/awscli-bundle.zip
$ unzip awscli-bundle.zip
Archive:  awscli-bundle.zip
  inflating: awscli-bundle/install
  inflating: awscli-bundle/packages/botocore-1.4.54.tar.gz
  inflating: awscli-bundle/packages/jmespath-0.9.0.tar.gz
  inflating: awscli-bundle/packages/simplejson-3.3.0.tar.gz
  inflating: awscli-bundle/packages/s3transfer-0.1.4.tar.gz
  inflating: awscli-bundle/packages/ordereddict-1.1.tar.gz
  inflating: awscli-bundle/packages/rsa-3.4.2.tar.gz
  inflating: awscli-bundle/packages/futures-3.0.5.tar.gz
  inflating: awscli-bundle/packages/awscli-1.10.64.tar.gz
  inflating: awscli-bundle/packages/docutils-0.12.tar.gz
  inflating: awscli-bundle/packages/colorama-0.3.7.zip
  inflating: awscli-bundle/packages/argparse-1.2.1.tar.gz
  inflating: awscli-bundle/packages/pyasn1-0.1.9.tar.gz
  inflating: awscli-bundle/packages/virtualenv-13.0.3.tar.gz
  inflating: awscli-bundle/packages/python-dateutil-2.5.3.tar.gz
  inflating: awscli-bundle/packages/six-1.10.0.tar.gz
$ cd awscli-bundle/packages/
$ ls -la | grep awscli
-rw-rw-rw-   1 jamesls  wheel   488607 Sep 13 16:38 awscli-1.10.64.tar.gz
```

Do you have a set of steps that demonstrate the issue?

**Comment by ferricoxide** [comment]
Unfortunately, no. All I know is that between the .62 and the .63 release, the first time I DL'ed .63, I ended up with both the .62 an .63 tarballs.

My general work-flow is that, each time I build an AMI, I do it from scratch. That is to say, I launch a fresh "bootstrap" AMI, download my AMI-generator scripts (that include a script that DLs and installs the latest AWS CLI), run those scripts to create an AMI within a chroot env built on a standalone EBS, register the snap the EBS, register the snap, destroy the bootstrap AMI.

The day I reported the issue was the first time since late 2014 that I ran into issues with the AWS CLI installer.
